### ISSUE TITLE
Compatibility with cookies exceptions

### ISSUE THREAD

**Issue by dlipovetsky** [hultmann/multifox#206]
Firefox can, by default, discard all cookies at the end of the session. It also allows the user to define exceptions, i.e. a whitelist of domains whose cookies should not be discarded at the end of the session.\* Before Firefox 37.0.1, cookies from whitelisted domains in a tab associated with an alternate (not default) Multifox profile would persist across sessions. Now, they are discarded.

For example, I can whitelist google.com and sign in to two Google accounts, one in the default profile and another in an alternate profile. When I restart the browser, I am no longer signed in to the Google account in the alternate profile. Prior to Firefox 37, I would have continued to be signed to both accounts.

I have confirmed this behavior using a fresh Firefox profile with no extensions except Multifox. 

I noticed that cookies for alternate Multifox profiles are stored under the ".multifox" domain, so I whitelisted it, but saw no change in behavior. I can investigate further, but thought I would first share the issue here.

[*] The relevant settings are under Preferences -> Privacy:
![firefox-privacy](https://cloud.githubusercontent.com/assets/3445370/7129554/dfde9a68-e216-11e4-9706-3bfd47210b3b.png)

**Comment by gatoruso** [comment]
I'm happy to debug, but not familiar with the Firefox extensions environment.

**Comment by dlipovetsky** [comment]
Sorry, accidentally closed the issue. If anyone uses cookies exceptions and Multifox, please leave a comment confirming or disconfirming this behavior. Thank you!

**Comment by hultmann** [comment]
Theoretically, you should be able to whitelist *.multifox cookies.

If it doesn't work, I think there is nothing Multifox could fix.

**Comment by dlipovetsky** [comment]
Thanks for your feedback! Contrary to my initial understanding, I now think the current behavior is by design, and correct, since the "keep me signed in" cookies are session cookies, and expire when Firefox is closed. I did learn something new: Firefox won't expire session cookies if they are [associated with a "pinned" tab](https://bugzilla.mozilla.org/show_bug.cgi?id=636399). I performed the same test as described in my initial post but with the accounts open in pinned tabs, and the logins persisted across browser restarts.
